File: totosuki/ABC/A/332.py
N, S, K = map(int, input().split())
data = [list(map(int, input().split())) for _ in range(N)]
rslt = 0
for d in data:
  p, q = d
  rslt += p * q

if rslt >= S:
  print(rslt)
else:
  print(rslt + K)

File: OCH_Tao/ABC/A/332.js
const input = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"));
const [, S, K] = input.shift().split(" ").map(x => parseInt(x, 10));
let cnt = 0;
for (const i of input) {
  const [P, Q] = i.split(" ").map(x => parseInt(x, 10));
  cnt += P * Q;
}
console.log(cnt >= S ? cnt : cnt + K);